Database/input-data/supplementary-source-id-info.yaml

```diff
@@ -97,6 +97,9 @@ MRI-JRA55-do-1-6-0:
   license_id: CC BY 4.0
 PCMDI-AMIP-1-1-9:
   license_id: CC BY 4.0
+PIK-CMIP-1-0-0:
+  license_id: CC BY 4.0
+  publication_status: expected August 2025
```

Comment:
After #264 is merged, you should be able to remove all of this. It should automatically be added by the data we get from the ESGF scrape